src/main/java/com/ibm/eventstreams/connect/mqsource/MQSourceConnector.java

```diff
@@ -107,7 +123,7 @@
     public static final String CONFIG_DOCUMENTATION_TOPIC = "The name of the target Kafka topic.";
     public static final String CONFIG_DISPLAY_TOPIC = "Target Kafka topic";
 
-    public static String VERSION = "1.1.0";
+    public static String VERSION = "1.2.0";
```

Comment:
I've been reviewing best practice for Maven versioning. This PR by itself will not form v1.2.0. Please use `1.2.0-SNAPSHOT` for all of the versions (you missed in instance in the Dockerfile I think).

Reply:
Correct, I missed that. Thanks for the hint!